src/styles/button.scss

```diff
@@ -3,7 +3,7 @@
   border-radius: var(--btn-border-radius);
   border-width: 0px;
   font-family: var(--font-family);
-  font-variation-settings: 'wght' var(--font-weight-normal);
+  font-weight: var(--font-weight-normal);
```

Comment:
Not sure if we are safe to replace `font-variation-settings` with plain `font-weight`. Here we use [variable font](https://rsms.me/inter/) which allows to set weights like `480`